assembl/static2/js/app/components/login/socialMediaLogin.jsx

```diff
@@ -34,7 +34,15 @@ export class SocialMedia extends React.Component {
         </h4>
         <ul>
           {this.props.providers.map((provider) => {
-            const providerName = convertNameToCssClass(provider.name);
+            let providerName = convertNameToCssClass(provider.name);
+            let classNameIcon = providerName;
+            let socialMediaName = provider.name;
+            // to implement google social media button
+            if (providerName === 'google-oauth2') {
+              providerName = 'google';
+              classNameIcon = 'gplus-1';
```

Comment:
Um...this may or may not be out of scope, but Google+ is now deprecated. Usage of the icons for this service may no longer make sense. The standard Google icon may be more appropriate. https://blog.google/technology/safety-security/expediting-changes-google-plus/

Reply:
@ayyazdaniaryan should create a new ticket for that one 👍